crm/modules/member/data.inc.php

```diff
@@ -415,8 +401,8 @@ function member_plan_save ($plan) {
     global $db_connect;
     $esc_name = mysqli_real_escape_string($db_connect, $plan['name']);
     $esc_price = mysqli_real_escape_string($db_connect, $plan['price']);
-    $esc_voting = mysqli_real_escape_string($db_connect, $plan['voting']);
-    $esc_active = mysqli_real_escape_string($db_connect, $plan['active']);
+    $esc_voting = mysqli_real_escape_string($db_connect, $plan['active']);
+    $esc_active = mysqli_real_escape_string($db_connect, $plan['enabled']);
```

Comment:
These variable names should probably be updated to reflect their new state.

Reply:
good catch, I'll look for more instances as well